core/cog-webkit-utils.c

```diff
@@ -69,61 +60,53 @@ load_error_page (WebKitWebView *web_view,
  * Returns: Whether other signal handlers should be skipped.
  */
 gboolean
-cog_handle_web_view_load_failed (WebKitWebView  *web_view,
```

Comment:
It looks like you ran `clang-format` on the whole file, is that so? We do not really encourage doing that and instead we prefer to slowly change only the parts of the code touched by changes towards the coding style checked by `clang-format`.

To do that, you would commit only the changes needed to improve the error page HTML and then run this to apply `clang-format` only on your changes:

```sh
data/check-style origin/master | git apply -p0
```

Reply:
Got it :). I've addressed these issues and updated the PR.